src/constant/global.ts

```diff
@@ -104,6 +104,10 @@ const apiStatusCodes = {
   INTERNAL_SERVER_ERROR: 500,
 };
 
+const underDevelopmentPage = {
+  projectsPage: /^\/projects(\/.*)?$/,
+};
+
```

Comment:
Constants can be named as IN_DEV_PAGES. Don't overcomplicate. Use an Array of all IN DEV Routes. If current page is found inside that array it's true. Keep the logic simple.